server/model/monitor.js

```diff
@@ -971,7 +982,13 @@ class Monitor extends BeanModel {
             } else if (bean.status === MAINTENANCE) {
                 log.warn("monitor", `Monitor #${this.id} '${this.name}': Under Maintenance | Type: ${this.type}`);
             } else {
-                log.warn("monitor", `Monitor #${this.id} '${this.name}': Failing: ${bean.msg} | Interval: ${beatInterval} seconds | Type: ${this.type} | Down Count: ${bean.downCount} | Resend Interval: ${this.resendInterval}`);
+                let groupText = "";
+                if (this.type !== "group" && this.parent !== "") {
+                    let parentName = await this.getParentName();
```

Comment:
Why not do it this way?

```suggestion
                    let parentName = await Monitor.getParent(this.id);
```

Reply:
continued in https://github.com/louislam/uptime-kuma/pull/4371/files#r1452821597